app/models/rule.rb

```diff
@@ -69,16 +70,20 @@ def canonical?
     (profiles & Profile.canonical).any?
   end
 
-  def self.from_openscap_parser(op_rule, benchmark_id: nil)
+  def self.from_openscap_parser(op_rule, benchmark_id: nil, group_id: nil)
     rule = find_or_initialize_by(ref_id: op_rule.id,
-                                 benchmark_id: benchmark_id)
+                                 benchmark_id: benchmark_id,
+                                 group_id: group_id)
 
     rule.op_source = op_rule
 
     rule.assign_attributes(title: op_rule.title,
                            description: op_rule.description,
                            rationale: op_rule.rationale,
                            severity: op_rule.severity,
+                           requires: op_rule.requires,
+                           conflicts: op_rule.conflicts,
+                           parent_type: op_rule.parent_type,
```

Comment:
Why is this one important? I feel like this is holding redundant information.

Reply:
I meant to reply to this earlier but I was originally thinking that it might be useful  in figuring out what the parent type is but it might not be necessary